static/open-specs/liveobjects.yaml

```diff
@@ -0,0 +1,641 @@
+
+paths:
+  /channels/{channelId}/objects:
+    get:
+      summary: List objects.
+      description: |
+        Fetches the list of objects stored on the channel.<br><br>
+
+        In the list API, data values represent a concrete piece of data (a number, a string, etc) or a reference to another object. The key in the data value indicates the type that you can expect to receive in the value.<br><br>
+
+        ```json
+          { "data": { "number" : 4 }}
+          { "data": { "string" : "Ably Pub/Sub" }}
+          { "data": { "boolean" : true }}
+          { "data": { "bytes": "TGl2ZU9iamVjdHMgaXMgYXdlc29tZQo=", "encoding": "base64" }}
+          { "data": {
+            "objectId":
+            "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000
+          }}
+        ```
+
+        <br>Maps are made of entries, which are a user-defined key (e.g. `myMapKey`) and a data value. Counters are data values containing a number.
+      tags:
+      parameters:
+        - name: channelId
+          in: path
+          required: true
+          description: The channel ID.
+          schema:
+            type: string
+            example: "my-app:likes:counter"
+        - name: values
+          in: query
+          required: false
+          description: Include the values in the response. Setting this to false returns a list of `objectIds` only.
+          schema:
+            type: boolean
+            default: false
+        - name: limit
+          in: query
+          required: false
+          description: Set the number of objects to be returned in each page.
+          schema:
+            type: integer
+        - name: cursor
+          in: query
+          required: false
+          description: The cursor used for pagination.
+          schema:
+            type: string
+        - name: metadata
+          in: query
+          required: false
+          description: Include object metadata in the response.
+          schema:
+            type: boolean
+            default: false
+      responses:
+        '200':
+          description: List of retrieved objects or object references.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: array
+                    items:
+                      type: object
+                      required:
+                        - objectId
+                      properties:
+                        objectId:
+                          type: string
+                        map:
+                          type: object
+                          properties:
+                            entries:
+                              type: object
+                              additionalProperties:
```

Comment:
perhaps we need some `example: ` keys in here?

Reply:
These won't render anywhere. They're shown in the line 202 of this commit as full block examples.